.travis.yml

```diff
@@ -1,9 +1,30 @@
+sudo: required
+
+dist: trusty
+
 language: java
 
-script: ./gradlew build
+install:
+  - wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+  - sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list'
+  - sudo apt-get update -q
+  - sudo apt-get install -q google-chrome-stable
+
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+script: 
+  - ./gradlew build
+  - cd src/main/resources/public
+  - npm run test
```

Comment:
Just looked through the recent job logs on base branch - haven't found any traces of js tests being executed - https://travis-ci.org/reportportal/service-ui/jobs/226474472
so we need it either in .travis.yml or inside gradle.build or inside grunt tasks